CHANGELOG.md

```diff
@@ -2,6 +2,7 @@
 
 ### 4.0.1 (Next)
 
+* [#223](https://github.com/alexa-js/alexa-app/issues/223): new card type available, AskForPermissionsConsent - [@ericblade]
```

Comment:
Lets make this consistent with the other changelog lines, capital first letter, a period in the end and a link to the github profile. Maybe:

```
* [#223](https://github.com/alexa-js/alexa-app/issues/223): Added support for `AskForPermissionsConsent` cards - [@ericblade](https://github.com/ericblade).
```